imagebuildah/stage_executor.go

```diff
@@ -1556,6 +1568,33 @@ func (s *StageExecutor) commit(ctx context.Context, createdBy string, emptyLayer
 		Manifest:              s.executor.manifest,
 		UnsetEnvs:             s.executor.unsetEnvs,
 	}
+	// generate build output
+	if s.executor.buildOutput != "" {
+		copierStripOpts := copier.GetOptions{}
+		if unshare.IsRootless() {
+			// Strip setuid,setgid,sticky and extended attributes
+			copierStripOpts.StripSetuidBit = true
+			copierStripOpts.StripSetgidBit = true
+			copierStripOpts.StripStickyBit = true
```

Comment:
Looking at the top-level `tmp` directory produced by buildkit, I think `StripStickyBit` needs to be left `false` always, and as a result not be part of a `ExtractRootfsOptions` struct.